### PR TITLE
If new sign up user has already pressed submit, don't send another request

### PR DESCRIPTION
**Pull request by hannahbergam**
This PR is meant to prevent honeybadger errors and multiple statsig events when users try to button mash their way to sign up success. If a user has already submitted once, this will check for that state and return before the submit actions are called again.

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/signUpFlow/FinishStudentAccount.tsx [hunk 1]
```diff
@@ -186,9 +186,12 @@ const FinishStudentAccount: React.FunctionComponent<{
   };
 
   const submitStudentAccount = async () => {
+    if (!!isSubmitting) {
```
Comment (TurnerRiley) [inline apps/src/signUpFlow/FinishStudentAccount.tsx]: Nit: since `isSubmitting` is set to `false` on load, I think this can just be `isSubmitting`

**Review by TurnerRiley — APPROVED** [review]
LGTM! One non-blocking nit

**Review by bethanyaconnor — APPROVED** [review]
I assume `isSubmitting` is already set to false if an error is returned? (I didn't look)

File: apps/src/signUpFlow/FinishStudentAccount.tsx (hunk 1 above)
Comment (hannahbergam) [reply apps/src/signUpFlow/FinishStudentAccount.tsx]: You right you right. updated!

**Comment by hannahbergam** [comment]
> I assume `isSubmitting` is already set to false if an error is returned? (I didn't look)

True true- we'll always have a bool. Updated!

**Pull request merged**